index.bs

```diff
@@ -223,9 +223,19 @@ its \[[Prototype]] an empty object, created as if by ObjectCreate(%ObjectPrototy
 
 <h3 id="logging">Logging methods</h3>
 
-<h4 id="assert" oldids="assert-condition-data,dom-console-assert" method for="console">assert(<var>condition</var>, ...<var>data</var>)</h4>
+<h4 id="assert" oldids="assert-condition-data,dom-console-assert" method for="console">assert(<var ignore=''>condition</var>, ...<var>data</var>)</h4>
```

Comment:
Trying to avoid a warning here:

```
WARNING: The following <var>s were only used once in the document:
  'condition'
If these are not typos, please add an ignore='' attribute to the <var>.
```